azure-pipelines/end-to-end-tests-dir/env-passthrough.ps1

```diff
@@ -5,13 +5,13 @@ if (-not $IsLinux -and -not $IsMacOS) {
     $env:_VCPKG_TEST_UNTRACKED = "b"
 
     $x = Run-Vcpkg "--overlay-triplets=$PSScriptRoot/../e2e_ports/env-passthrough" env "echo %_VCPKG_TEST_TRACKED% %_VCPKG_TEST_TRACKED2% %_VCPKG_TEST_UNTRACKED% %_VCPKG_TEST_UNTRACKED2%"
-    if ($x -ne "%_VCPKG_TEST_TRACKED% %_VCPKG_TEST_TRACKED2% %_VCPKG_TEST_UNTRACKED% %_VCPKG_TEST_UNTRACKED2%")
+    if ($x -ne "%_VCPKG_TEST_TRACKED% %_VCPKG_TEST_TRACKED2% %_VCPKG_TEST_UNTRACKED% %_VCPKG_TEST_UNTRACKED2%`r`n")

```

Comment:
Will this work on Linux? We should run the end-to-end tests on our Linux agent as well.

Reply:
We do. But this test starts with `if (-not $IsLinux -and -not $IsMacOS) {`. (The test wouldn't work on Linux or MacOS because they don't use `%`s)